SUMMARY.md

```diff
@@ -180,6 +180,7 @@
 ## Contributors
 
 * [Contributors](contributors/contributors/README.md)
+  * [Contribution Guidelines and Code of Conduct](contributors/contributors/contribution-guidelines.md)
   * [Mattermost Community](contributors/contributors/community.md)
   * [Community Systems](contributors/contributors/community-systems.md)
```

Comment:
Let's move it to below "Community Systems" in the hierarchy

Reply:
Done! https://github.com/mattermost/mattermost-handbook/pull/526/commits/1e2f469b6de766ef0389e2c70f490436f160fa53